docs/howto/openstack/neutron/del-network.md

```diff
@@ -0,0 +1,553 @@
+router, you start by disconnecting the subnet from the router. Then, 
+you will move on to deleting the subnet and the network, and after 
+that, you can finish up with deleting the router.
+
+=== "{{gui}}"
+    Go to the _Subnets_ tab of the `carmacks` network, and click the gray 
+    notepad-and-pen icon (at the left of the red circle-with-trashcan icon).
+        
+    ![Network subnets](assets/del-net/shot-03.png)
+    
+    A vertical pane titled _Modify Subnet_ will slide over from the 
+    right-hand side of the page. Pay attention to the _Router Connections_ 
+    section. You will notice an active connection to the router. Click the 
+    red circle-with-line-over-chainlink icon to deactivate the connection, 
+    effectively disconnecting the subnet from the router.
+        
+    ![Disconnect subnet](assets/del-net/shot-04.png)
+        
+    A pop-up window will appear, asking if you really want to go ahead with 
+    the disconnection. Just click the red _Yes, Remove interface_ button.
+        
+    ![Remove interface](assets/del-net/shot-05.png)
+        
+    After disconnecting the subnet, click the red circle-and-trashcan icon 
+    to delete it. Once more, a pop-up will appear asking for confirmation. 
+    Click the red _Yes, Delete_ button.
+        
+    ![Delete subnet](assets/del-net/shot-06.png)
+        
+    As soon as you delete the subnet, in the _Subnets_ tab you will see the 
+    message _No subnets found_.
+        
+    ![No subnets](assets/del-net/shot-07.png)
+        
+    You can now delete the network. Click the three-dot icon (right-hand 
+    side of the network row) and select _Delete Network_.
+        
+    ![Delete Carmacks](assets/del-net/shot-08.png)
+        
+    Of course, you will have to confirm this action. Clicking the red _Yes, 
+    Delete_ button is enough.
+        
+    ![Confirm network delete](assets/del-net/shot-09.png)
+        
+    After deleting the network, it will not be on the list of all available 
+    networks.
+        
+    ![List of networks](assets/del-net/shot-10.png)
+        
+    There's still that router lying around, and if you have no use for it, 
+    go to the _Routers_ page to delete it. In the vertical pane on the 
+    left, expand the _Networking_ section and click on _Routers_. In the 
+    central pane of the page, you will see all routers in all regions you 
+    have access to.
+    
+    ![All routers](assets/del-net/shot-11.png)
+        
+    Click the red three-dot icon of the router you wish to delete and 
+    select _Delete Router_. A pop-up will appear asking for confirmation, 
+    so click the red _Yes, Delete_ button.
+    
+    ![Confirm router delete](assets/del-net/shot-12.png)
+    
+    After successfully deleting the router, there will be no trace of it in 
+    the list of all routers.
+        
+    ![Router is gone](assets/del-net/shot-13.png)
+=== "OpenStack CLI"
+    Let us first take a look at all available subnets:
```

Comment:
You can use the imperative here. Thus, "First, take a look" etc.

Reply:
I sure can :)